http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -157,8 +160,8 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
             _ <- publisherQueue.offer(
               CreateAzureRuntimeMessage(savedRuntime.id,
                                         workspaceId,
-                                        relayNamespace,
                                         storageContainer.resourceId,
+                                        landingZoneResourcesOpt,
```

Comment:
shall we just pass relayNamespace from the LZ directly here? it seems the change will be a lot smaller that way...given back leo doesn't really need other info from the LZ either

Reply:
i was thinking this too, it's a lot to pass everything... but now I saw from rob we do need the vnet and compute subnet too, so I'm leaning towards sending the whole object-

Reply:
sounds good...